cmd/root.go

```diff
@@ -120,6 +123,16 @@ func helperFlagFormatter(fs *pflag.FlagSet) string {
 // preExecHook is executed after the command line is parsed but
 // before the command's handler is executed
 func preExecHook(cmd *cobra.Command, args []string) {
+	logLocation, _ := cmd.Flags().GetString("log-loc")
+	_, err := os.Open(logLocation + "/fsoc.log")
+	if err == nil {
+		os.Remove(logLocation + "/fsoc.log")
+	}
+	file, _ := os.Create(logLocation + "/fsoc.log")
+	log.SetHandler(multi.New(cli.New(os.Stderr), json.New(file)))
+	if logLocation == "/dev/null" {
+		log.SetHandler(multi.New(cli.New(os.Stderr)))
```

Comment:
Will this achieve the desired effect of:
- all events are logged in the file (DebugLevel)
- stderr shows the selected level (InfoLevel if -v, WarnLevel otherwise)

Reply:
If i understand the question correctly, then I believe so